bitnami/ghost/templates/svc.yaml

```diff
@@ -27,5 +21,7 @@ spec:
       {{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.nodePorts.http)))}}
       nodePort: {{ .Values.service.nodePorts.http }}
```

Comment:
It is important to set the nodePort to null in case the service is of type ClusterIP. There were support cases that complained that helm upgrade would fail if you changed a deployment from LoadBalancer to ClusterIP. Check the WordPress chart svc.yaml for an example.

Reply:
done